images/nvidia-device-plugin/config/main.tf

```diff
@@ -9,11 +9,25 @@ variable "extra_packages" {
   default     = ["nvidia-device-plugin"]
 }
 
-data "apko_config" "this" {
-  config_contents = file("${path.module}/latest.apko.yaml")
-  extra_packages  = var.extra_packages
+module "accts" {
+  source = "../../../tflib/accts"
+  run-as = 0
 }
 
 output "config" {
-  value = jsonencode(data.apko_config.this.config)
+  value = jsonencode({
+    contents = {
+      packages = var.extra_packages
+    }
+    accounts = module.accts.block
+    entrypoint = {
+      command = "/usr/bin/nvidia-device-plugin"
+    }
+    environment = {
+      "NVIDIA_VISIBLE_DEVICES" : "all",
+      "NVIDIA_DRIVER_CAPABILITIES" : "utility",
+      "NVIDIA_DISABLE_REQUIRE" : "true",
+      "NVIDIA_MIG_MONITOR_DEVICES" : "all",
```

Comment:
just curious, where does this env come from?

i only find reference of it coming from the helm chart.
https://github.com/search?q=repo%3ANVIDIA%2Fk8s-device-plugin%20NVIDIA_MIG_MONITOR_DEVICES&type=code

Reply:
Ah yep you're right, I was going off of a running container but this seems to be set by the helm chart. Will remove so it can be configured properly

Reply:
Done